Source/SubtitleParser.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "SubtitleParser.g.h"
+#include <FFmpegMediaSource.h>
+
+namespace winrt::FFmpegInteropX::implementation
+{
+    struct SubtitleParser : SubtitleParserT<SubtitleParser>
+    {
+        SubtitleParser() = default;
+
+        SubtitleParser(winrt::com_ptr<winrt::FFmpegInteropX::implementation::FFmpegMediaSource> interopMSS)
+        {
+            this->interopMSS = interopMSS;
+        }
+
+        static winrt::Windows::Foundation::IAsyncOperation<winrt::FFmpegInteropX::SubtitleParser> ReadSubtitleAsync(winrt::Windows::Storage::Streams::IRandomAccessStream stream, hstring streamName, winrt::FFmpegInteropX::MediaSourceConfig config, winrt::Windows::Media::Core::VideoStreamDescriptor videoDescriptor, uint64_t windowId, bool useHdr);
+        static winrt::Windows::Foundation::IAsyncOperation<winrt::FFmpegInteropX::SubtitleParser> ReadSubtitleAsync(winrt::Windows::Storage::Streams::IRandomAccessStream stream);
+        winrt::FFmpegInteropX::SubtitleStreamInfo SubtitleTrack();
```

Comment:
Many of those namespace prefixes aren't needed, 
If it is your general convention to always use full qualification, then it's fine of course. It's just moving the more relevant parts of lines off-screen.